package/victoria-metrics.service

```diff
@@ -1,15 +1,15 @@
 [Unit]
-Description=VictoriaMetrics is a fast, cost-effective and scalable time series database.
+Description=High-performance, cost-effective and scalable time series database, long-term remote storage for Prometheus
+After=network.target
 
 [Service]
 Type=simple
 StartLimitBurst=5
 StartLimitInterval=0
-Restart=on-failure
-RestartSec=1
-ExecStart=/usr/sbin/victoria-metrics -storageDataPath /var/lib/victoria-metrics-data -retentionPeriod 12
+Restart=on-abort
+ExecStart=/usr/local/bin/victoriametrics -storageDataPath=/var/lib/victoria-metrics-data start
```

Comment:
@patsevanton , what does `start` mean in the end of this line?

Reply:
@valyala changed to 
```
ExecStart=/usr/local/bin/victoriametrics -storageDataPath=/var/lib/victoria-metrics-data
```

Reply:
@kmansoft , then remove the trailing `start` from here